core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/ExoPlayerForwarder.java

```diff
@@ -58,8 +58,9 @@ public void bind(NoPlayer.CompletionListener completionListener, NoPlayer.StateC
         exoPlayerEventListener.add(new OnCompletionStateChangedForwarder(stateChangedListener));
     }
 
-    public void bind(NoPlayer.ErrorListener errorListener) {
+    public void bind(NoPlayer.ErrorListener errorListener, NoPlayer.ErrorListener internalErrorListener) {
         exoPlayerEventListener.add(new PlayerOnErrorForwarder(errorListener));
+        exoPlayerEventListener.add(new PlayerOnErrorForwarder(internalErrorListener));
```

Comment:
the `internalErrorListener` will be called after `errorListener` because `ExoPlayerEventListener` adds all the listeners to a `List`